pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -102,7 +102,7 @@ def _modify_sw_link_clicked(self, label, uri):
     def _get_sw_needs_text(self, required_space, sw_space, auto_swap):
         tooltip = _("Please wait... software metadata still loading.")
 
-        if isinstance(self.payload, LiveImagePayload):
+        if self.payload.type in (PAYLOAD_TYPE_LIVE_OS, PAYLOAD_TYPE_LIVE_IMAGE):
```

Comment:
I'm thinking there should be smarter solution than checking to these two.

One idea is to create a constant in CORE:
``PAYLOAD_TYPE_LIVE = (PAYLOAD_TYPE_LIVE_OS, PAYLOAD_TYPE_LIVE_IMAGE)``

Reply:
I almost agreed, but after reflecting on this, I think spelling out the multiple options is better. The test uses a different operator for multiple values. If you created a "merged types constant" then you would still have to use a different operator, but you could not tell easily why.

Actually - is there a chance this could get out of hand? Such as, testing for even more types? Two is fine, I guess, but more would be weird.

Reply:
Fixed with `PAYLOAD_LIVE_TYPES`.